geonode_mapstore_client/client/js/components/ZoomTo/ZoomTo.jsx

```diff
@@ -32,11 +32,20 @@ const ZoomTo = ({
             } else {
                 bounds = aBounds;
             }
-            map.getView().fit(bounds, {
-                size: map.getSize(),
-                duration: 300,
-                nearest: true
-            });
+            // ensure when map is contained in tab and
+            // container needs update followed by zoom to extent
+            setTimeout(()=> {
+                map.updateSize();
+            }, 200);
+
+            setTimeout(()=> {
+                map.getView().fit(bounds, {
+                    size: map.getSize(),
+                    duration: 300,
+                    nearest: true
+                });
+            }, 300);
+
```

Comment:
This is not needed, the problem about tabs was related to the animation. I would prefer to apply animation={false} on Tabs instead of this workaround

Reply:
Good to know. Should have saved me some time. Thanks 👍